src/Propel/Runtime/Propel.php

```diff
@@ -145,6 +146,29 @@ public static function getServiceContainer(): ServiceContainerInterface
         return self::$serviceContainer;
     }
 
+    /**
+     * Returns the service container if it is an instance of
+     * StandardServiceContainer or throws an error if another service container
+     * is used.
+     *
+     * This method allows type-safe access to methods of
+     * StandardServiceContainer, it provides no other advantage over
+     * {@link Propel::getServiceContainer()}
+     *
+     * @throws \Propel\Runtime\Exception\PropelException
+     *
+     * @return \Propel\Runtime\ServiceContainer\StandardServiceContainer
+     */
+    public static function getStandardServiceContainer(): StandardServiceContainer
```

Comment:
It seems that due to the method logic it would be better to call it `getStandardServiceContainerOrFail()` just because this method throws an exception.

Reply:
We only add orFail() though if we need to keep a nullable alternative. This seems not like that case.